tests/backup/namespace_labelled_backup_test.go

```diff
@@ -0,0 +1,197 @@
+package tests
```

Comment:
Change the file name to backup_namespace_labelled_test.go

Reply:
done